readme.md

```diff
@@ -26,5 +41,30 @@ USE_PYETHEREUM_DEVELOP=1 python setup.py develop
 
 ### Install development tool
 ```shell
+pip install --upgrade pip;
 pip install -r dev_requirements.txt
 ```
+
+### Troubleshooting
+
+##### macOS
+
+* SSL dependency error on macOS
+
+  * Problem: Error occurs when running `USE_PYETHEREUM_DEVELOP=1 python setup.py develop`
+
+    ```
+    scrypt-1.2.0/libcperciva/crypto/crypto_aes.c:6:10: fatal error: 'openssl/aes.h' file not found
+    #include <openssl/aes.h>
+             ^~~~~~~~~~~~~~~
+    1 error generated.
+    error: Setup script exited with error: command 'clang' failed with exit status 1
+    ```
+
+  * Solution: Set SSL flags to valid directories
+
+    ```
+    brew update; brew upgrade openssl; which openssl;
+    export LDFLAGS="-L/usr/local/opt/openssl/lib";
+    export CPPFLAGS="-I/usr/local/opt/openssl/include"
+    ```
```

Comment:
This is helpful. Thank you 👍 
btw. is `which openssl;` just used to confirm `openssl` is installed?

Reply:
I've added `which openssl` because it shows the directory where the OpenSSL binary executable is installed after they install it with Homebrew. That way users can verify where OpenSSL is installed and tweak the directory names that they add as values of those flags. 
For instance on my computer it returns the following, which shows me that OpenSSL is installed in `/usr/local/opt/openssl/`, and if I check the contents of that directory with `ls -lha /usr/local/opt/openssl/` then it shows me that it contains subdirectories `lib` and `include`:
```shell
$ which openssl
/usr/local/opt/openssl/bin/openssl
```